cEP-0010.md

```diff
@@ -0,0 +1,136 @@
+# coala Configuration
+
+| Metadata     |                                                |
+| ------------ |------------------------------------------------|
+| cEP          | 10                                             |
+| Version      | 1.0                                            |
+| Title        | Convert bears to use aspects                   |
+| Authors      | Asnel Christian Ngoulla <ngoullasob@gmail.com> |
+| Status       | Implementation Due                             |
+| Type         | Feature                                        |
+
+## Abstract
+
+This document describes how `aspects` will be implemented and bears' results
+annotated with them.
+
+This cEP also provides a list of the `aspects` to be implemented with their
+correspponding `tastes` or `settings`.
+
+## Introduction
+
+An `aspect` is an analysis we want to run over a piece of code. As stated in
+[cEP](https://github.com/coala/cEPs/blob/master/cEP-0005.md), one of the main
+problems coala faces is its configuration due to the huge number of bears
+(capable of analyzing several aspects of code) sometimes highly configurable
+(with numerous settings) it offers; whereby the user has to be aware of bear
```

Comment:
Better *"... due to the huge number of bears it offers, which are capable of analyzing several aspects of code and are sometimes highly configurable with numerous settings; whereby the user has to be aware of which bear does what..."*